engine/src/gfx/particle.cpp

```diff
@@ -76,9 +76,9 @@ void ParticleTrail::ChangeMax( unsigned int max )
 
 
 
-static inline void UpdateColor( std::vector< QVector, aligned_allocator<QVector> > &vloc,
-                                const std::vector< Vector, aligned_allocator<Vector> > &vvel,
-                                std::vector< GFXColor, aligned_allocator<GFXColor> > &vcol,
+static inline void UpdateColor( std::vector< QVector > &vloc,
+                                const std::vector< Vector > &vvel,
+                                std::vector< GFXColor > &vcol,
```

Comment:
Um. I think there was a reason that `aligned_allocator`s were being used here. Possibly an important reason. I suggest that we not change this if we can help it. (Though changing its iterator references to `auto`, further down, is fine by me.)

Reply:
I removed them as the types being used were not compliant with the requirements for the `aligned_allocator`. I'm not familiar enough with building a type that would be. Happy to have a PR against my branch to make it work properly or we can address it again in another PR.